pkg/minishift/openshift/openshift.go

```diff
@@ -269,6 +274,8 @@ func getLocalConfigFile(target string) string {
 			return "/var/lib/minishift/openshift.local.config/node-localhost/node-config.yaml"
 		}
 		return "/var/lib/minishift/base/node/node-config.yaml"
+	case "kube":
```

Comment:
I feel we are going overboard by fixing it in minishift because there are futher changes to the config.yaml files in 3.11. Ideally this should be fixed in OpenShift side.  
Also I am wondering whether we should call it only kube or kubeapiserver as it is admission webhook is a kube api server thing.

Reply:
@LalatenduMohanty we always have to provide a way to user to patch the openshift config or kube config even webhook become the default. We are overboard this because that's the change we have to accommodate from cluster up side.